.gitignore

```diff
@@ -131,3 +131,4 @@ dmypy.json
 .vim
 *.db
 **/.DS_STORE
+.vscode/settings.json
```

Comment:
similarly-ish, just ignore .vscode/* - it'll save hassle in future if someone else using vsc has to do it

Reply:
Done